organize/models.py

```diff
@@ -129,7 +129,8 @@ def __str__(self):
 
     def save(self, *args, **kwargs):
         if not self.latlng:
-            self.latlng = get_coordinates_for_city(self.city, self.get_country_display())
+            latlng_coord = get_coordinates_for_city(self.city, self.get_country_display())
+            self.latlng = latlng_coord[:30]  # Truncate to 30 characters to match the database field limit
```

Comment:
If we get values like this `(-74.80111590000001, -74.80111590000001) of which each value is 19 characters, we would truncate this and have the first value remain 19 characters while the second value becomes -74.80111 is 9 characters long. It would be better to format the values in the `get_coordinates_for_city()` and limit the number of decimals places for each so that they are of equal length, making consideration for the possible `-` in the values. The function is here  https://github.com/DjangoGirls/djangogirls/blob/main/core/utils.py#L18 in `core/utils.py`.

Thanks for your work on this.

Reply:
Thank you very much for the review and letting me know @amakarudze 
I Got it. Let me work on this and I will get back soon.

Reply:
I have made those changes. Please let me know if any changes are required.

Reply:
Thanks, sorry for not getting back to you earlier on. Just have one more comment below.

Reply:
One more change, can you truncate to 7 decimal places, tests are failing because the values in the tests are set to 7 decimal places while your changes truncate to 6 decimal places. Truncating to 7 decimal places should fix this. Also use double quotes for the `f` string to address ruff issues.

Reply:
Yes sure. Will do. Thank you